SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -1018,16 +1023,19 @@ void FSpatialGDKEditorToolbarModule::OpenInspectorURL()
 	}
 }
 
-void FSpatialGDKEditorToolbarModule::LaunchInspectorWebpageButtonClicked()
+void FSpatialGDKEditorToolbarModule::StartInspectorProcess(TFunction<void()> OnReady)
 {
 	const USpatialGDKEditorSettings* SpatialGDKEditorSettings = GetDefault<USpatialGDKEditorSettings>();
 	const FString InspectorVersion = SpatialGDKEditorSettings->GetInspectorVersion();
 
-	AsyncTask(ENamedThreads::AnyBackgroundThreadNormalTask, [this, InspectorVersion] {
+	AsyncTask(ENamedThreads::AnyBackgroundThreadNormalTask, [this, InspectorVersion, OnReady] {
 		if (InspectorProcess && InspectorProcess->Update())
 		{
-			// We already have an inspector running. Just open the URL.
-			OpenInspectorURL();
+			// We already have an inspector process running. Call ready callback if any.
+			if (OnReady)
```

Comment:
should this just be if (OnReady) or should it be if (OnReady != nullptr)

miron votes != nullptr

Reply:
(but it seems we just use if (x) everywhere?)